config/environments/production.rb

```diff
@@ -43,7 +43,6 @@
     'X-XSS-Protection' => '0',
     'X-Frame-Options' => 'SAMEORIGIN',
     'X-Content-Type-Options' => 'nosniff',
-    'X-Robots-Tag' => 'noindex',
```

Comment:
Do we want this line to be `'X-Robots-Tag' => 'none' if ENV['WORKSPACE'].eql?('production')`

Reply:
Yes, so we are consistent

Reply:
Done